lib/sycamore/sycamore/tests/unit/transforms/test_merge_elements.py

```diff
@@ -169,7 +169,6 @@ class TestGreedySectionMerger:
                 },
                 {
                     "type": "Image",
-                    "text_representation": "image1 on page 2 before text5",
```

Comment:
I thought you were going to add "summary" : {"summary" : "image1 on page 2 before text5", ..}
Having only an image without a summary will test the new if branch, but there is no test to test the image with summary merging. Please add a second image with summary

Reply:
added another test for with image summary which I forgot to add earlier.